clio-client/src/main/scala/org/broadinstitute/clio/client/webclient/ClioWebClient.scala

```diff
@@ -88,4 +90,21 @@ class ClioWebClient(clioHost: String, clioPort: Int, useHttps: Boolean)(
       ).addHeader(Authorization(credentials = OAuth2BearerToken(bearerToken)))
     )
   }
+
+  def getResponseAsJson(httpResponse: Future[HttpResponse]): Future[Json] = {
+    httpResponse.flatMap(Unmarshal(_).to[Json])
+  }
+
+  def ensureOkResponse(
+    httpResponse: Future[HttpResponse]
+  ): Future[HttpResponse] = {
+    httpResponse.flatMap(
+      r =>
+        r.status match {
+          case StatusCodes.OK => Future.successful(r)
```

Comment:
This might be too restrictive; better to have an if-else checking if `r.status.isSuccess()`.